app/controllers/catalog_controller.rb

```diff
@@ -1243,19 +1237,19 @@ def set_resource_action(st)
     end
   end
 
-  def st_set_record_vars(st)
+  def common_record_vars(st)
```

Comment:
it sets the common vars for both atomic & composite service templates, which is why i didn't think st_ prefix should be added.